data/human/human missions.txt

```diff
@@ -7411,3 +7411,1828 @@ mission "Adelita 5"
+			`	Light reflects off the tears that cling to Timothy's eyes as he tells the assembled crowd that, for him, that was your greatest moment, your most heroic action, that nothing you've done, or ever will do, compares to that simple act of mercy. And that this, to him, alone, is what makes you great.`
+			`	From here his story shifts to your selfless offer to help save Rand, even though they could promise no reward, of your stunning victory over Greenrock, and your daring liberation of the fleet on the ground. He ends with you triumphant, and Rand saved.`
+			`	The assembled guests stand, clap and cheer. They look at you, at your seat of honor, and raise their glasses to you. Eventually as the cheers and hoots die down, the Governor declares you a Hero of Rand, and thanks you for your service.`
+			choice
+				`	(Say a few words.)`
+				`	(Mingle with the crowd.)`
+					goto mingle
+				`	(Skip the party.)`
+					goto skip
+
+			`	You walk up to the podium, past the governor and turn to address the assembled guests.`
+			choice
+				`	(Humbly thank them.)`
+					goto thankthem
+				`	(Tell them it is well deserved.)`
+					goto deserved
+				`	(Dress them down for being capitalist pigs.)`
+
+			`	You address the crowd. You compliment them on how well dressed they all are, how lavish this mansion is, how wealthy they all appear. They clap, and laugh at your apt observations. But, then you talk about how poor, dirty, and shabby things are outside of this mansion. With its fake gravity, and fake greenery. You declare you want no praise from them, no flattery, and no accolades. There's a million Heroes of Rand, outside these baroque walls, toiling away in dirty, dangerous mines. All for what? So that these company-bosses and fat-cat politicians can pat each other on the back and live in obscene luxury in the middle of a squalid world? You rail, you rant, you break them down, and you turn the crowd against you. Angry people are shouting back at you now, boos and hisses are rising up from all sides of the room. And yet, you see a glimmer of respect in the eyes of servants. The waiters, the attendants, the assistants, they clench their fists and nod as your speech goes on. They hear you. They agree with you. A plate crashes into the podium. Another slams into the pillar nearby you. The crowd bulging toward you. The esteemed guests with murder in their eyes.`
```

Comment:
```suggestion
			`	You address the crowd. You compliment them on how well dressed they all are, how lavish this mansion is, how wealthy they all appear. They clap, and laugh at your apt observations. But, then you talk about how poor, dirty, and shabby things are outside of this mansion, with its fake gravity, and fake greenery. You declare you want no praise from them, no flattery, and no accolades. There are a million Heroes of Rand, outside these baroque walls, toiling away in dirty, dangerous mines. All for what? So that these company-bosses and fat-cat politicians can pat each other on the back and live in obscene luxury in the middle of a squalid world? You rail, you rant, you break them down, and you turn the crowd against you. Angry people are shouting back at you now, boos and hisses are rising up from all sides of the room. And yet, you see a glimmer of respect in the eyes of servants. The waiters, the attendants, the assistants, they clench their fists and nod as your speech goes on. They hear you. They agree with you. A plate crashes into the podium. Another slams into the pillar nearby you. The crowd bulging toward you, esteemed guests with murder in their eyes.`
```